test/quantization/test_qat.py

```diff
@@ -1449,7 +1468,16 @@ def test_fake_quantize_per_token_vs_convert(self):
     @unittest.skipIf(
         not TORCH_VERSION_AT_LEAST_2_4, "skipping when torch version is 2.4 or lower"
     )
-    def test_qat_8da4w_prepare_vs_convert(self):
+    def test_fake_quantize_per_token_vs_convert_fp32(self):
+        self._test_fake_quantize_per_token_vs_convert(torch.float32)
+
+    @unittest.skipIf(
+        not TORCH_VERSION_AT_LEAST_2_4, "skipping when torch version is 2.4 or lower"
+    )
+    def test_fake_quantize_per_token_vs_convert_bf16(self):
```

Comment:
nit: do we have float16 as well? also can probably use parametrization for these